python/cuml/dask/ensemble/randomforestregressor.py

```diff
@@ -285,7 +285,7 @@ def _fit(model, X_df_list, y_df_list, r):
 
     @staticmethod
     def _predict(model, X, r):
-        return model.predict(X)
+        return model.predict(X).copy_to_host()
```

Comment:
Right now dask RF is doing the aggregation of results in the CPU and therefore returning CPU data, would be good to add a github issue to have the option of doing this in the GPU and return GPU memory